x-pack/docs/en/security/securing-communications/security-minimal-setup.asciidoc

```diff
@@ -51,14 +51,15 @@ discovery.type: single-node
 [[security-create-builtin-users]]
 ==== Set passwords for built-in users
 
-To communicate with the cluster, you must configure a username for the built-in
-users. Unless you enable anonymous access, all requests that don’t include a
-username and password are rejected.
+To communicate with your cluster, you must configure a password for
+the `elastic` and `kibana_system` built-in users. Unless you enable anonymous
+access (not recommended), all requests that don’t include a username and
+password are rejected.
```

Comment:
I think `username and password` is too specific. Might want to replace it with "credentials" since we support more than just username password.